parquet-hadoop/src/test/java/org/apache/parquet/hadoop/TestBloomEncryption.java

```diff
@@ -0,0 +1,313 @@
+/*
```

Comment:
I guess, this test and the one for column indexes were created by copy-pasting the original tests and adding the encryption. Since we already use parameterized testing in both I would suggest keeping the original tests and adding another dimension for _plain_/_encrypted_ or similar. This way we would have 4 runs for each tests: (V1 with plain), (V1 with encryption), (V2 with plain) and (V2 with encryption).

Reply:
this will be a part of https://github.com/apache/parquet-mr/pull/782 (it will also remove the TestBloomEncryption.java and TestColumnIndexEncryption.java files).

Reply:
Thank you, @gszadovszky , for the suggestion. We've added it to #782 .